### ISSUE TITLE
Duplicate entries on github actions update

### ISSUE THREAD

**Issue by dharrigan** [liquidz/antq#246]
Hi!

`Clojure CLI version 1.11.1.1435`
`com.github.liquidz/antq {:mvn/version "2.8.1169"}`

When running against a repo that has `.github/workflows/*.yml`, duplicate entries are created if (I think) there is an additional line between steps.

Before (notice the additional line between the `build` and `publish` steps):

```
---

name: Production

on:
  push:
  workflow_dispatch:

jobs:
  build:
    name: Build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3


  publish:
    name: Publish
    runs-on: ubuntu-latest
    steps:
      - uses: actions/cache@v3
        id: restore-build
        with:
          path: ./*
          key: ${{ github.sha }}-${{ github.run_number }}-${{ github.run_attempt }}
```

Executing: `clojure -X:antq`

after:

```
---

name: Production

on:
  push:
  workflow_dispatch:

jobs:
  build:
    name: Build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4


  publish:
    name: Publish
    runs-on: ubuntu-latest
    steps:
      - uses: actions/cache@v3
        id: restore-build
        with:
          path: ./*
          key: ${{ github.sha }}-${{ github.run_number }}-${{ github.run_attempt }}

  publish:
    name: Publish
    runs-on: ubuntu-latest
    steps:
      - uses: actions/cache@v3
        id: restore-build
        with:
          path: ./*
          key: ${{ github.sha }}-${{ github.run_number }}-${{ github.run_attempt }}
```

I've created a repo where you can reproduce this issue: https://github.com/dharrigan/antq-issue-246

-=david=-

**Comment by liquidz** [comment]
@dharrigan Thanks for your reporting!
I could reproduce the problem. I'll fix it :)

**Comment by liquidz** [comment]
Memo:  this is a problem on the [rewrite-indented](https://github.com/liquidz/rewrite-indented) side.

**Comment by liquidz** [comment]
@dharrigan I've fixed this problem in [dev](https://github.com/liquidz/antq/tree/dev) branch.
Could you try dev branch?

**Comment by dharrigan** [comment]
Hi,

Works perfectly. ありがとう :-)

-=david=-

**Comment by liquidz** [comment]
@dharrigan Thanks for your confirmation!
I just released v2.8.1173 :)
